pyspark/README.md

```diff
@@ -0,0 +1,20 @@
+<!---
+ Copyright 2022-2023 Alibaba Group Holding Limited.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+
+# GraphAr PySpark
```

Comment:
The README should include information like:
- Introduction
- Dependency
- How to build and use graphar_pyspark